### DISCUSSION TITLE
[Feature Request] wiki refactor

### DISCUSSION THREAD

**Discussion by pjz** [community/community#23020] [New to GitHub]
<p>This started out as a request to add wiki editing to the activity timeline sot hat my cohorts who spend a lot of time updating and refining and adding content to the wiki have their efforts shown.</p>
<p>Then I searched for ‘wiki’ on this board. Oh boy.  Lots of requests… most of which are for features that normal repos already have.</p>
<p>So I’ll make a different suggestion instead: Just like repos can have a ‘gh-pages’ mode, allow repos to have a ‘wiki mode’ - that would automatically get them: 1) activity tracking 2) PRs 3) better access controls … to name just a few things.   ‘wiki mode’ means (afaict) to show Home.md by default and pay attention to _* files (like _Footer.md and _Sidebar.md).</p>
<p>Alternately, instead of putting the entire repo into ‘wiki mode’ for all pages, have a ‘view as wiki’ link that leads to a slightly different URL tree that is that repo, but rendered as a wiki.</p>
<p>I think that will solve a lot of problems in a lot nicer way than trying to port a bunch of features to work on special ‘wiki repos’.  Autoconversion from the current state of affairs should be fairly straightforward since they’re already repos - just not ‘full fledged’ ones.</p>
<p>Thanks for your consideration,</p>
<p>  --pj</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/pjz">@pjz</a> ,</p>
<p>Thanks for the feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive.   I’ve taken your suggestion and passed it along to the appropriate teams.</p>
<p>I don’t have an ETA on when or if this request will be implemented but this is in good hands.</p>

**Reply by danielorbach** [reply]
Hi @nyahbhinghiPrincess ,

Can you update about any intention to implement this? My organisation would like to have our Wiki on GitHub, but the current feature-set is not sufficient and seems to not get updates often.
